pkg/cloudfoundry/Authentication.go

```diff
@@ -26,11 +30,11 @@ func LoginCheck(options LoginOptions) (bool, error) {
 	var cfCheckLoginScript = append([]string{"api", options.CfAPIEndpoint}, options.CfAPIOpts...)
 
 	var cfLoginBytes bytes.Buffer
-	c.Stdout(&cfLoginBytes)
+	_c.Stdout(&cfLoginBytes)
```

Comment:
Does this make `stdout` inmutable for LoginCheck function?

Reply:
No. I already spottet that too. We have to reset the old stdout afterwards. But at the moment it is not possible to retrieve the current stdout via our wrapper. Needs to be added. At least the underlying [cmd](https://golang.org/src/os/exec/exec.go?s=15990:16036#L115) allow to get the currently set stream(s).

The point is: we have to ensure that the "cmdRunner" is not permanently changed inside the func. I had this already. As a result logging output stopped after calling login. Of course also for the subsequent `cf deploy`.

Reply:
I would propose to finalize on discussion in #1760 . This probably solves this whole discussion and we can leave the whole function out